### PR TITLE
added centos_msg pod config

### PR DESCRIPTION
**Pull request by sakalosj**
tested - pod is running in stg environment

### CODE REVIEW

File: openshift/packit-service-centosmsg.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  name: packit-service-centosmsg
+spec:
+  template:
+    metadata:
+      labels:
+        name: packit-service-centosmsg
+        app: packit
+    spec:
+      volumes:
+        - name: packit-secrets
```
Comment (csomh) [inline openshift/packit-service-centosmsg.yml.j2]: While this will work, I'm a little bit worried of storing unrelated stuff in the same secret.

For one, this way the `packit-service-centosmsg` pod will be coupled to the `packit-service` one through this shared secret.

Two: this opens the door for `packit-service` to have access to centos messaging secrets, which is absolutely not required.

File: vars/template.yml [hunk 2]
```diff
@@ -47,6 +47,9 @@ without_fedmsg: true
 # don't deploy redis-commander by default
 without_redis_commander: true
 
+# don't deploy centosmsg dc; good for local testing
+without_centosmsg: true
```
Comment (csomh) [inline vars/template.yml]: I think this should be also defined and set to some default value in `playbooks/deploy.yml#L32`

File: openshift/packit-service-centosmsg.yml.j2 (hunk 1 above)
Comment (sakalosj) [reply openshift/packit-service-centosmsg.yml.j2]: ok will create issue for this, not right time for this now

Comment (csomh) [reply openshift/packit-service-centosmsg.yml.j2]: Why not? You could create a new secret for the centos messaging secrets now and use that. You definitely shouldn't start to split up everything right now.

File: vars/template.yml (hunk 2 above)
Comment (sakalosj) [reply vars/template.yml]: fixed

Comment (sakalosj) [reply vars/template.yml]: fixed

File: openshift/packit-service-centosmsg.yml.j2 (hunk 1 above)
Comment (sakalosj) [reply openshift/packit-service-centosmsg.yml.j2]: thought it will require more time - fixed

**Comment by sakalosj** [comment]
> LGTM, which repo is the `listen-to-centos-messaging` code in?

@jpopelka 
https://github.com/packit-service/packit-service-centosmsg
